scripts/bumpSdkVersion.ts

```diff
@@ -5,6 +5,7 @@ import path from "path";
 import semver from "semver";
```

Comment:
I am amused that our bump version script is written in a language that doesn't exist for any sdk :D